velox/docs/develop/types.rst

```diff
@@ -145,3 +145,38 @@ Supported range of milliseconds is [0xFFF8000000000000L, 0x7FFFFFFFFFFFF]
 (or [-69387-04-22T03:45:14.752, 73326-09-11T20:14:45.247]). The low 12 bits
 store timezone ID. Supported range of timezone ID is [1, 1680].
 The definition of timezone IDs can be found in ``TimeZoneDatabase.cpp``.
+
+Spark Types
+~~~~~~~~~~~~
+The data types in Spark have some semantic differences compared to those in 
```

Comment:
perhaps, add a link to Spark's documentation: https://spark.apache.org/docs/latest/sql-ref-datatypes.html

Reply:
Updated, thanks.